gm4_liquid_minecarts/data/liquid_minecarts/functions/init_liquid.mcfunction

```diff
@@ -0,0 +1,19 @@
+#@s = empty liquid minecart next to a liquid tank pointed at it containing liquid
+#run from liquid_minecarts:try_to_unload
+
+data modify entity @s CustomName set from block ~ ~ ~ CustomName
+data modify entity @e[type=armor_stand,tag=gm4_liquid_minecart_display,limit=1,sort=nearest] HandItems[0] set from entity @e[type=armor_stand,tag=gm4_liquid_tank_display,limit=1,sort=nearest] ArmorItems[3]

```

Comment:
What is the point of having the liquid texture copied to the hand item if you cannot see it? If its not intended to be seen, maybe moving it to the chest slot would avoid forcing the player to render the skull.

Reply:
I guess there was no reason except that I could see it worked. I can move it to the chest slot!